tables-test-fixtures/src/main/java/com/linkedin/openhouse/tablestest/SpringH2TestApplication.java

```diff
@@ -24,12 +24,14 @@
     basePackages = {
       "com.linkedin.openhouse.tables.api",
       "com.linkedin.openhouse.tables.audit",
+      "com.linkedin.openhouse.tables.toggle",
       "com.linkedin.openhouse.tables.authorization",
       "com.linkedin.openhouse.tables.dto.mapper",
       "com.linkedin.openhouse.tables.utils",
       "com.linkedin.openhouse.tables.controller",
       "com.linkedin.openhouse.tables.services",
       "com.linkedin.openhouse.tables.config",
+      "com.linkedin.openhouse.tables.toggle.repository",
```

Comment:
is tables-test-fixtures working fine after `;MODE=MYSQL` changes? is it compatible with H2?

Reply:
Lets keep an eye out for tables-test-fixtures!